CMakeLists.txt

```diff
@@ -16,7 +16,6 @@ option(TRITON_BUILD_TUTORIALS "Build C++ Triton tutorials" ON)
 option(TRITON_BUILD_PYTHON_MODULE "Build Python Triton bindings" OFF)
 
 # Default build for this branch is ROCm support
-option(TRITON_USE_ROCM "Build with ROCm/AMDGPU support" ON)
```

Comment:
This should be removed to prevent AMDGPU specific code to build by default.

This implies that the TRITON_USE_ROCM env var must be defined.

Reply:
Actually you can just set the default option value to "OFF", so keeping the ability to specify it wih "-D" option to cmake, without modifying an environment.
Nit: perhaps the describing comment should be correct also.